pyuvdata/uvcal/tests/test_uvcal.py

```diff
@@ -2024,6 +2024,101 @@ def test_reorder_jones_errors(gain_data):
         calobj.reorder_jones(order=np.arange(2))
 
 
+@pytest.mark.filterwarnings("ignore:Combined Jones elements are not evenly spaced")
+@pytest.mark.parametrize("add_type", ["ant", "time", "freq", "jones"])
+@pytest.mark.parametrize("sort_type", ["ant", "time", "freq", "jones"])
+@pytest.mark.parametrize("future_shapes", [True, False])
+def test_add_different_sorting(gain_data, add_type, sort_type, future_shapes):
```

Comment:
So I worry this test might not actually poke at the underlying issue that was discovered, which appeared when you had overlapping-but-differently-ordered data. As far as I can tell, it seems like all of these tests split the data in half, and reorder the halves before recombination. It would be good to have at least some subset of these have that overlap present.

Reply:
Hmm, I tried to structure this test in the same way as the test you wrote for #1105. I don't think that was related to actually overlapping data (as in data for exactly the same blt, freq, pol) or we would have correctly errored. I thought it was when you had an axis that was not involved in the sum (so had all the same values on the objects) that was differently ordered.

Reply:
Ahh, yes -- sorry, I missed the fact that you've got the equivalent of a double loop here. Clearly been too long since I've written a test...